### ISSUE TITLE
odd symbol table behavior

### ISSUE THREAD

**Issue by bruceravel** [xraypy/xraylarch#106]
At line 195 of the feffrunner.py file in my recent PR (#105), I have
```python
+def initializeLarchPlugin(_larch=None):
   """initialize _xafs._feff_executable"""
   _larch.symtable.set_symbol('_xafs._feff_executable', ['/usr/local/bin/feff6',])
```
I did this following the example of setting the [`_xray/_materials` symbol](https://github.com/xraypy/xraylarch/blob/master/plugins/xray/materials.py#L155)

I found this did not work when the value of the symbol was a string.  Hence the use of a one-element list.  Here's what happens when that symbol is set to a string.
```
larch> show _xafs._feff_executable
  File "/usr/local/lib/python2.7/dist-packages/larch/__init__.py", line 37, in wrapper
    return fcn(*args, **keywords)
  File "/usr/local/share/larch/plugins/std/show.py", line 54, in _show
    group = symtable._lookup(sym, create=False)file <stdin>, line 0
NameError: '/usr/local/bin/feff6' is not defined
    show(_xafs._feff_executable)
```
That's a rather confusing error message.

It is certainly possible that the problem here is that I am not understanding something about the symbol table....

**Comment by newville** [comment]
@bruceravel  Hmm, I'm not 100% sure.  it might be that

```python
     _larch.symtable.set_symbol('Group.Member', thing)

```
will not work if the Group does not exist (it should work like mkdir -p, but may not).   But '_xafs` should exist, I think when this is run....  Maybe try

```python
  if _larch.symtable.has_group('_xafs'):
     xafsgroup = _larch.symtable.get_group('_xafs')
  else:
     xafsgroup = larch.Group()
  xafsgroup.feff_executable = '/usr/bin/feff6'
  _larch.symtable._xafs = xafsgroup
```

I'll look into this more too...

**Comment by bruceravel** [comment]
>     _larch.symtable.set_symbol('Group.Member', thing)
>
> will not work if the Group does not exist (it should work like mkdir -p, but may not). 
> But '_xafs` should exist,

If `thing` is a string, I see the behavior I reported.  If thing is a list, it works as expected.  That seems to indicate that the type of `thing` is the issue.

**Comment by newville** [comment]
@bruceravel  hold on, I read the original message wrong.  The symbol `_xafs._feff_executable` is there whether it is a string or not.   The error message is coming from `show()`.    

```python
 larch> show(_xafs._feff_executable)
```

is asking to show the string value held in `_xafs._feff_executable`.   What `show()` does with a non-Group object is slightly under-defined -  right now `show()` treats a string as a symbol name and tries to look it up.   Here, it could find no symbol named '/usr/local/bin/feff6'.  I'll fix it to just print out the string.

**Comment by bruceravel** [comment]
Apparently, my role in the larch universe is to hammer on `show()` :smiley: 

Once you have fixed this, I would like to make some changes to my feffrunner plugin, including to the symbol in question.  Perhaps you could reject #105 and let me make a new PR later ...?

**Comment by bruceravel** [comment]
Fixed in 9554c47022c4d85f39e45dfc929777ae64a1c1b9
